### DISCUSSION TITLE
Add non-scored or weirdly-scored reviews

### DISCUSSION THREAD

**Discussion by cosmo0** [benbarden/switchscores#51] [Ideas]
I know you don't really want to add non-scored reviews because you want users to easily find "the best" games.

But some sites test a lot of smaller games without scoring or with a "weird" scale:

- https://nindienexus.net/category/reviews/ (now moved to https://medium.com/nindienexus )
- https://nintendoscene.com/category/reviews/
- https://ladiesgamers.com/category/review-2/switch-game-reviews-impressions/

Listing them might not help for the "ranking games" part, but I still think it would help users if they had easier access to more reviews, even if they have to be listed in a different "non-scored reviews" list, and not counted in the games lists (or differently).

**Comment by benbarden** [comment]
I think this fits with the idea of partner content, which could include unscored reviews, user guides, and features. Similar to the "From the blog" section: https://www.switchscores.com/games/4660/sakuna-of-rice-and-ruin

**Reply by cosmo0** [reply]
I've talked with NindieSpotlight and they might be interested if they could have a basic "red/yellow/green" score for quick reviews. I find the idea interesting.

**Reply by benbarden** [reply]
We need something numeric if there's a score. To get an average score you can't do 8 + 7 + 6 + "green" :)

It is possible to convert ratings to a number internally, but the partner would have to be happy with what we do. For example a site might use Excellent/Great/Good/Fair/Poor on their site, but we could convert them to 10/8/6/4/2. We can do that without the score needing to appear on the partner site. However, it would appear on Switch Scores. I could hide it for certain sites, but it wouldn't be hard to work it out.

**Reply by benbarden** [reply]
The alternative is to allow quick reviews without scores, which just means they wouldn't be included in the overall rankings. That's better than nothing for games with no reviews as at least you get a comment. Though it's better to have a score too, as otherwise we will have games that look like they need 4 reviews to be ranked (rather than the current 3) - which complicates things quite a bit.

**Reply by cosmo0** [reply]
I meant that a green/yellow/red review could be considered non-scored but still have a "quick glance" label. But maybe that would complicate and confuse things too much.

I agree non-scored reviews shouldn't count with scored ones. Maybe something like this?

>[ 2/3 ]  
>reviews (+1)
